packages/dx-react-chart/demos/featured/olympic-games.md

```diff
@@ -0,0 +1,5 @@
+# React Chart Olympic Games
+
+This demo shows the Chart component that displays a bar series with customized axes and bars.
```

Comment:
Can we change this text?
For example `This demo shows the Chart component that displays a medal amount of the Olympic games with a` [BarSeries](https://devexpress.github.io/devextreme-reactive/react/chart/docs/reference/bar-series/)